aws/resource_aws_ses_identity_notification.go

```diff
@@ -0,0 +1,137 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ses"
+	"github.com/hashicorp/terraform/helper/schema"
+)
+
+func resourceAwsSesNotification() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsSesNotificationSet,
+		Read:   resourceAwsSesNotificationRead,
+		Update: resourceAwsSesNotificationSet,
+		Delete: resourceAwsSesNotificationDelete,
+
+		Schema: map[string]*schema.Schema{
+			"topic_arn": &schema.Schema{
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateArn,
+			},
+
+			"notification_type": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateNotificationType,
+			},
+
+			"identity": &schema.Schema{
+				Type:         schema.TypeString,
+				Required:     true,
+				ValidateFunc: validateIdentity,
+			},
+		},
+	}
+}
+
+func resourceAwsSesNotificationSet(d *schema.ResourceData, meta interface{}) error {
```

Comment:
We need to call `d.SetId()` in here so we can reference the resource identifier in the read function and testing. I would recommend creating a two part ID such as `d.SetId(fmt.Sprintf("%s|%s", identity, notification))`

This will also allow us to later implement importing.

Reply:
Done.